testutil/compose/auto.go

```diff
@@ -125,6 +125,11 @@ func Auto(ctx context.Context, conf AutoConfig) error {
 		defer cancel()
 	}
 
+	// Build docker-compose services before executing docker-compose up.
+	if err = execBuild(ctx, conf.Dir); err != nil {
```

Comment:
Found out from logs that this steps longer for the first test which is included in alerts timeout